addons/medical/functions/fnc_actionCheckPulseLocal.sqf

```diff
@@ -46,9 +46,14 @@ if (_heartRate > 1.0) then {
     };
 };
 
+if (_selectionName in ["hand_l","hand_r"] && [_unit, _selectionName] call FUNC(hasTourniquetAppliedTo)) then {
```

Comment:
add here lazyEelevation that could Improve the Performance
``if (_selectionName in ["hand_l","hand_r"] && {[_unit, _selectionName] call FUNC(hasTourniquetAppliedTo)}) then {`